packages/core/ios/LottieReactNative.xcodeproj/project.pbxproj

```diff
@@ -118,6 +118,7 @@
 				11FA5C4D1C4A1296003AC2EE /* Sources */,
 				11FA5C4E1C4A1296003AC2EE /* Frameworks */,
 				11FA5C4F1C4A1296003AC2EE /* CopyFiles */,
+				423561892A827E2800D7D5AB /* Run SwiftLint */,
```

Comment:
I don't think this is a good idea to run this on users' environments. This should only be done on our side.

Reply:
Let's do it in CI.

Reply:
Makes sense. Removed under [fbbe7dd](https://github.com/lottie-react-native/lottie-react-native/pull/1085/commits/fbbe7ddb5f08e54b6f89ea935e485eb8f3747a08). Gonna add a CI step after the PR is merged